src/py/flwr/superexec/exec_servicer.py

```diff
@@ -35,9 +35,10 @@
 class ExecServicer(exec_pb2_grpc.ExecServicer):
     """SuperExec API servicer."""
 
-    def __init__(self, executor: Executor) -> None:
+    def __init__(self, executor: Executor, config: Optional[Dict[str, str]]) -> None:
```

Comment:
If we don't pass the config to `start_run` any more, this won't be necessary (b/c `executor` already contains the config)